janusgraph-dist/src/assembly/static/cassandra/conf/cassandra.yaml

```diff
@@ -31,26 +37,23 @@ cluster_name: 'Cassandra testing instance'
 # If you leave this unspecified, Cassandra will use the default of 1 token for legacy compatibility,
 # and will use the initial_token as described below.
 #
-# Specifying initial_token will override this setting.
+# Specifying initial_token will override this setting on the node's initial start,
+# on subsequent starts, this setting will apply even if initial token is set.
 #
 # If you already have a cluster with 1 token per node, and wish to migrate to 
-# multiple tokens per node, see https://wiki.apache.org/cassandra/Operations
-#num_tokens: 256
+# multiple tokens per node, see http://wiki.apache.org/cassandra/Operations
+num_tokens: 256
```

Comment:
I think the next link would be better because this one is broken. 
http://cassandra.apache.org/doc/latest/configuration/cassandra_config_file.html#num-tokens

Reply:
same here